Assets/Scripts/Augment/AugmentImplementations/Pan.cs

```diff
@@ -22,30 +26,48 @@ void Start()
         {
             gameObject.transform.localScale = new Vector3(2f, 2f, 2f);
             hitboxCollider.gameObject.layer = hitBoxLayer;
+            explosionColliders.SetActive(false);
             return;
         }
         panModel.localScale = new Vector3(2.5f, 2.5f, 2.5f);
         panModel.localPosition = new Vector3(panModel.localPosition.x, panModel.localPosition.y -0.3f, panModel.localPosition.z);
-        gunController.Player.inputManager.onSelect += TryTrickJump;
+        if (gunController.Player.inputManager)
+            gunController.Player.inputManager.onSelect += TryTrickJump;
         hitboxCollider.enabled = false;
         playerMovement = gunController.Player.GetComponent<PlayerMovement>();
-
+        var health = gunController.Player.GetComponent<HealthController>();
+        gunController.Player.GunOrigin.GetComponentsInChildren<PanHitbox>()
+            .ToList().ForEach(box =>  box.health = health);
+        if (playerMovement)
+            playerMovement.OnMove += TryPanSkateBoost;
     }
 
     private void TryTrickJump(InputAction.CallbackContext ctx)
     {
         if (playerMovement)
-            if (playerMovement.StateIsAir)
+            if (playerMovement.StateIsAir && playerMovement.IsCrouching)
```

Comment:
at this point the `if` pyramid looks a little odd

Reply:
Hehehe, I concede that the pyramid should go, I will miss it tho 🫡